lib/Push.php

```diff
@@ -157,6 +166,25 @@ private function parseXmlResponse()
                 }
             }
 
+            // set Executives
+            if (isset($xml->Transaction, $xml->Transaction->Customer->Company->Executive)) {
+                $executivesCount = $xml->Transaction->Customer->Company->Executive->count();
```

Comment:
If you change the order of lines 171 and 172 you can simplify
```suggestion
                $xmlExecutive = $xml->Transaction->Customer->Company->Executive;
                $executivesCount = $xmlExecutive->count();
```